docs-source/spring/content/setup/_index.md

```diff
@@ -121,3 +124,22 @@ Oracle Backend as a Service for Spring Cloud setup will install the following co
     <!-- spellchecker-disable -->
     {{< img name="oci-stack-apply-logs" size="large" lazy=false >}}
     <!-- spellchecker-enable -->
+
+## Setup the local workstation
+
+1. Install Local Tools
+
+    Before you set up the [Kubernetes access](./cluster-access), you must have already done the following:
+
+    - generated an API signing key pair
+    - added the public key value of the API signing key pair to the User Settings for your username
+    - installed and configured the Oracle Cloud Infrastructure CLI (version 2.6.4 or later)
+    - installed and configured the Kubernetes command-line tool (kubectl)
+
+    To use `kubectl` on your local machine, you will need to have the [OCI CLI installed and configured](https://docs.oracle.com/en-us/iaas/Content/API/Concepts/cliconcepts.htm).
+
+2. Install Oracle Backend as a Service for Spring Cloud command-line
+
+    The Oracle Backend as a Service for Spring Cloud command-line tool, `oracle-spring`, is available for Linux and Mac systems. Download the binary you want from the [Releases](https://github.com/oracle/microservices-datadriven/releases/tag/OBAAS-1.0.0) page and add it to your PATH environment variable.  You may like to rename the binary to remove the suffix.
+
+    If you're environment is a Linux or Mac machine you need to execute `chmod +x` on the downloaded binary. Also if your environment is a Mac you need execute the following command `sudo xattr -r -d com.apple.quarantine <downloaded-file>` otherwise will you get a security warning and the CLI will not work.
```

Comment:
your, not you're
don't say "you need to execute" just tell them to do it, and don't use the word "execute" 

You can just say "On MacOS, run this command to (do whatever)" etc

Reply:
still has incorrect word "you're" and unacceptable word "execute"

Reply:
replaced all occurrences when possible.